sdv/metadata/multi_table.py

```diff
@@ -35,6 +36,65 @@ def _set_metadata_dict(self, metadata):
         for relationship in metadata.get('relationships', []):
             self._relationships.append(relationship)
 
+    def visualize(self, show_table_details, show_relationship_labels, output_filepath=None):
```

Comment:
Should `show_table_details` and `show_relationship_labels` be optional, and be able to run `digraph = metadata.visualize()` ?